poetry.toml

```diff
@@ -1,7 +1,3 @@
-# This file is copied into our docker images to globally configure the poetry tool
-
 [virtualenvs]
-# We're in a docker container, there's no need for virtualenvs
-# One should still specify "ENV PIP_USER yes" to let poetry know
-# install packages as --user so they show up in ~/.local
-create = false
+create = true
+in-project = true
```

Comment:
are we sure this is what we want? Is this evaluated on the host machine? or in the Dockerfile build?

Reply:
this is strictly used for aladdin development, it could be removed all together and leave it up to the devs global config, I don't have a strong opinion so I might just remove it